assets/js/src/form_editor/store/map_form_data_after_loading.jsx

```diff
@@ -17,9 +17,15 @@ export default function mapFormDataAfterLoading(data) {
           styles: { ...defaults.popupForm.styles, ...data.settings.form_placement?.popup?.styles },
           posts: {
             all: data.settings.form_placement?.popup?.posts?.all === '1',
+            selected: Array.isArray(data.settings.form_placement?.popup?.posts?.selected)
```

Comment:
Shouldn't we add this property to mapper test? I think it could be handy if we decide to refactor the mapper in the future.